daemon/logging/logging_outlets.go

```diff
@@ -142,7 +143,30 @@ func (o *SyslogOutlet) WriteEntry(entry logger.Entry) error {
 		if now.Sub(o.lastConnectAttempt) < o.RetryInterval {
 			return nil // not an error toward logger
 		}
-		o.writer, err = syslog.New(syslog.LOG_LOCAL0, "zrepl")
+		fac := syslog.LOG_LOCAL0
+		switch o.Facility {
+			case "kern":     fac = syslog.LOG_KERN
+			case "user":     fac = syslog.LOG_USER
+			case "mail":     fac = syslog.LOG_MAIL
+			case "daemon":   fac = syslog.LOG_DAEMON
+			case "auth":     fac = syslog.LOG_AUTH
+			case "syslog":   fac = syslog.LOG_SYSLOG
+			case "lpr":      fac = syslog.LOG_LPR
+			case "news":     fac = syslog.LOG_NEWS
+			case "uucp":     fac = syslog.LOG_UUCP
+			case "cron":     fac = syslog.LOG_CRON
+			case "authpriv": fac = syslog.LOG_AUTHPRIV
+			case "ftp":      fac = syslog.LOG_FTP
+			case "local0":   fac = syslog.LOG_LOCAL0
+			case "local1":   fac = syslog.LOG_LOCAL1
+			case "local2":   fac = syslog.LOG_LOCAL2
+			case "local3":   fac = syslog.LOG_LOCAL3
+			case "local4":   fac = syslog.LOG_LOCAL4
+			case "local5":   fac = syslog.LOG_LOCAL5
+			case "local6":   fac = syslog.LOG_LOCAL6
+			case "local7":   fac = syslog.LOG_LOCAL7
+		}
```

Comment:
this should be done in `config.go` `parseSyslogOutlet`, the `SyslogOutlet.Facility` member should be `syslog.Priority` ( https://golang.org/pkg/log/syslog/#Priority )